Employee.m

```diff
@@ -38,31 +40,33 @@ -(id)copyWithZone:(NSZone *)zone{
     return employee;
 }
 
-@end
 
 
-//-(NSNumber *)employeeNumber{
-//    return _employeeNumber;
-//}
-//-(void)setEmployeeNumber:(NSNumber *)employeeNumber{
-//    _employeeNumber = employeeNumber;
-//}
-//
-//
-//
-//-(NSNumber *)yearsEmployed{
-//    return _yearsEmployed;
-//}
-//-(void)setYearsEmployed:(NSNumber *)yearsEmployed{
-//    _yearsEmployed = yearsEmployed;
-//}
+-(NSNumber *)employeeNumber{
```

Comment:
For clarity when looking at your code, you might put all of your getters together, and all of your setters together.